src/shared/logger/activation.ts

```diff
@@ -127,7 +128,7 @@ export function makeLogger(
     if (opts.useDebugConsole) {
         logger.logToDebugConsole()
     }
-    if (opts.useConsoleLog) {
+    if (opts.useBrowserConsole && isInBrowser()) {
```

Comment:
I don't understand why isInBrowser() is needed here. We have extensionWeb.ts which can control things, instead of adding special-cases to inner components like this.

Reply:
Updated the code, let me know if that is better.